<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>451</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBoxOP1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_LongOp">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Дизъюнкция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Конъюкция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Инверсия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Импликация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эквивалентность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исключающее или</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxOP2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Логические операции</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
